test_src/sljitTest.c

```diff
@@ -11602,6 +11604,7 @@ static void test92(void)
 
 	label = sljit_emit_label(compiler);
 	sljit_emit_op2(compiler, SLJIT_ADD, SLJIT_R1, 0, SLJIT_S0, 0, SLJIT_IMM, 9 * sizeof(sljit_sw));
+	sljit_emit_op1(compiler, SLJIT_MOV, SLJIT_S2, 0, SLJIT_IMM, WCONST(0xAAAAAAAAAAAAAAAAu, 0xAAAAAAAAu));
```

Comment:
What does this test here?

Reply:
That the load doesn't spill over its specified register width, or even if it does, that the rest of the transaction is safe from it.

Indeed, might be worth adding to the spec what is the relevant width specially for U8 and U16.

Reply:
The load overwrites the whole register. The upper bits are zero. You have already made a SLJIT_R2=-1 test above for this, so maybe you can also use -1 here.

Reply:
I got confused by my own change (though it was yours), agree it is useless, and indeed I think it (the oe using -1) might had been introduced by mistake from my own debugging.

Would it make sense to keep some of these enabled under some DEBUG mode?, will remove them otherwise.